### PR TITLE
Update bookings-in-outlook.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the description for EWS Enabled

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8563

File: microsoft-365/bookings/bookings-in-outlook.md [hunk 1]
```diff
@@ -110,7 +110,7 @@ Use the **Get-OrganizationConfig** and **Set-OrganizationConfig** commands to fi
 
 Use the **Get-CASMailbox** and **Set-CASMailbox** commands to check user status and turn Bookings in Outlook on or off for individual users in your organization.
 
-1. Check individual’s EwsApplicationAccessPolicy by running the following command:
+1. Check individual’s EWS control access by running the following command:
```
Comment (JohanFreelancer9) [inline microsoft-365/bookings/bookings-in-outlook.md]:
```suggestion
1. Check the individual’s EWS control access by running the following command:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**